engine/src/main/scala/cromwell/services/MetadataServiceActor.scala

```diff
@@ -57,45 +37,31 @@ object MetadataServiceActor {
   case class MetadataLookupResponse(query: MetadataQuery, eventList: Seq[MetadataEvent]) extends MetadataServiceResponse
   case class MetadataServiceKeyLookupFailed(query: MetadataQuery, reason: Throwable) extends MetadataServiceResponse
 
-  def props(serviceConfig: Config, globalConfig: Config) = {
-    Props(MetadataServiceActor(serviceConfig, globalConfig))
+  def props(dataAccess: DataAccess, serviceConfig: Config, globalConfig: Config) = {
```

Comment:
If we have different dataAccesses for engine and this service, this could be in-memory while the engine might not be.  Then again it seems like almost anything interesting would be stored via this service.

Is there a change I should make here for this?